doc/sphinx/changes.rst

```diff
@@ -8,6 +8,618 @@ Recent changes
+- **Added:**
+  :cmd:`Search` understands modifier ``in`` as an alias of ``inside``
+  (`#15139 <https://github.com/coq/coq/pull/15139>`_,
+  fixes `#14930 <https://github.com/coq/coq/issues/14930>`_,
+  by Gaëtan Gilbert).
+  This is intended to ease transition for ssreflect Search users.
+- **Fixed:** interaction of Program's obligation state and modules and
+  sections: obligations started in a parent module or section are not
+  available to be solved until the submodules and subsections are
+  closed (`#14780 <https://github.com/coq/coq/pull/14780>`_, fixes
+  `#14446 <https://github.com/coq/coq/issues/14446>`_, by Gaëtan
+  Gilbert).
+- **Fixed:**
+  :cmd:`Eval` and :cmd:`Compute` now beta-iota-simplify the type
+  of the result, like :cmd:`Check` does
+  (`#14901 <https://github.com/coq/coq/pull/14901>`_,
+  fixes `#14899 <https://github.com/coq/coq/issues/14899>`_,
+  by Hugo Herbelin)
+
+Command-line tools
+^^^^^^^^^^^^^^^^^^
+
+- **Changed:**
+  Coqdoc options ``--coqlib`` and ``--coqlib_path`` have been renamed
+  to ``--coqlib_url`` and ``--coqlib`` to make them more consistent with
+  flags used by other Coq executables
+  (`#14059 <https://github.com/coq/coq/pull/14059>`_,
+  by Emilio Jesus Gallego Arias).
+- **Changed:**
+  Syntax of `_CoqProject` files: `-arg` is now handled by :ref:`coq_makefile
+  <coq_makefile>` and not by `make`. Unquoted `#` now start line comments.
+  (`#14558 <https://github.com/coq/coq/pull/14558>`_,
+  by Columbus240, with help from Jim Fehrle and Enrico Tassi).
+- **Removed:**
+  These options of :ref:`coq_makefile <coq_makefile>`: `-extra`, `-extra-phony`,
+  `-custom`, `-no-install`, `-install`, `-no-opt`, `-byte`.
+  Support for subdirectories is also removed.
+  (`#14558 <https://github.com/coq/coq/pull/14558>`_,
+  by Columbus240, with help from Jim Fehrle and Enrico Tassi).
+- **Added**:
+  :ref:`coq_makefile <coq_makefile>` now takes the `-docroot` option as alternative to the
+  `INSTALLCOQDOCROOT` variable
+  (`#14558 <https://github.com/coq/coq/pull/14558>`_,
+  by Columbus240, with help from Jim Fehrle and Enrico Tassi).
+- **Changed:**
+  :cmd:`Require` now selects files whose logical name
+  exactly matches the required name, making it possible to unambiguously select
+  a given file: if several :n:`-Q` or :n:`-R` options bind the same
+  logical name to a different file, the option appearing last on the
+  command line takes precedence.  Moreover, it is now an error to
+  require a file using a partial logical name which does not resolve
+  to a non-ambiguous path (`#14718
+  <https://github.com/coq/coq/pull/14718>`_, by Hugo Herbelin).
+
+- **Fixed:**
+  Various `coqdep` issues with the `From` clause of :cmd:`Require` and
+  a few inconsistencies between `coqdep` and `coqc` disambiguation
+  of :cmd:`Require`
+  (`#14718 <https://github.com/coq/coq/pull/14718>`_,
+  fixes `#11631 <https://github.com/coq/coq/issues/11631>`_
+  and `#14539 <https://github.com/coq/coq/issues/14539>`_,
```

Comment:
These entries are not correctly ordered. We can notice that the first one has the colon outside the boldface instead of inside, so that may be related. But didn't the updated script raise some warning in this case?

Reply:
Actually, they come from only two changelog files initially, so this is related to my remark in your other PR. But still, we got a `Removed` inserted in the middle of many `Changed`.

Reply:
fixed